drivers/SmartThings/zwave-thermostat/profiles/base-thermostat-8to20C.yml

```diff
@@ -0,0 +1,32 @@
+name: base-thermostat-8to20C
+components:
+- id: main
+  capabilities:
+  - id: temperatureMeasurement
+    version: 1
+  - id: thermostatHeatingSetpoint
+    version: 1
+    configuration:
+          - values:
+            key: "temperature.value"
```

Comment:
This needs to be `heatingSetpoint.value`. The capability attribute name is `heatingSetpoint`, not `temperature`